### PR TITLE
Appeal-19613: Development and Domo Seeds and Test data

### PR DESCRIPTION
**Pull request by pamatyatake2**
…caseflow VhaCamo

Resolves 

- [Appeal-19614](https://vajira.max.gov/browse/APPEALS-19614)
- [Appeal-19615](https://vajira.max.gov/browse/APPEALS-19615)
- [Appeal-19616](https://vajira.max.gov/browse/APPEALS-19616)
- [19623](https://vajira.max.gov/browse/APPEALS-19623)
- [19624](https://vajira.max.gov/browse/APPEALS-19624)
- [19625](https://vajira.max.gov/browse/APPEALS-19628)
- [19627](https://vajira.max.gov/browse/APPEALS-19627)
- [19628](https://vajira.max.gov/browse/APPEALS-19628)
### Description
**Added Seeds for populating VhaCamo Organization and VhaCaregiver  queue to have 5 of each:** 
  - Assigned
  - In Progress
  - Completed
- added 3 new methods to seeds/tasks.rb 
- added 1 new method to factories/task.rb
- 
**Adding Seeds for VHA Program Offices**
  - Assigned
  - In Progress
  - Ready for Review
  - On Hold
  - Completed

**List of VHA Program Offices:**
  - Community Care - Payment Operations Management
  - Community Care - Veteran and Family Members Program
  - Member Services - Health Eligibility Center
  - Member Services - Beneficiary Travel
  - Prosthetics

- added 2 new methods to seeds/tasks.rb 
- added 1 new method to factories/task.rb

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] vha-camo should have pre-populated queues in all 3 tabs when switched to organization view.
- [ ] vha-caregiver should have pre-populated queue in all 3 tabs when switched to organization view.
### Testing Plan
1. Switch user to be either CAMOUser or CAMOADMIN 
2.  Go to /organizations/vha-camo
3.  All three tabs should have task prepopulated.
4.  `bundle exec rails runner scripts/run_seed_data_file.rb db/seeds/users.rb` -> this might cause an error since users are already in the db, you can either run all the seeds or comment out certain potion and just run this script to test.
5. ` bundle exec rails runner scripts/run_seed_data_file.rb db/seeds/veterans_health_administration.rb`
6. When clicking on the Case Detail link all task should be redirected to the detail page. 

image

- [ ] For higher-risk changes: [Deploy the custom branch to UAT to test](https://github.com/department-of-veterans-affairs/appeals-deployment/wiki/Applications---Deploy-Custom-Branch-to-UAT)

### User Facing Changes
 - [ ] Screenshots of UI changes added to PR & Original Issue

 BEFORE|AFTER
 ---|---

### Code Documentation Updates
- [ ] Add or update code comments at the top of the class, module, and/or component.

### Storybook Story
*For Frontend (Presentationa) Components*
* [ ] Add a [Storybook](https://github.com/department-of-veterans-affairs/caseflow/wiki/Documenting-React-Components-with-Storybook) file alongside the component file (e.g. create `MyComponent.stories.js` alongside `MyComponent.jsx`)
* [ ] Give it a title that reflects the component's location within the overall Caseflow hierarchy
* [ ] Write a separate story (within the same file) for each discrete variation of the component

### CODE REVIEW

File: db/seeds/tasks.rb [hunk 1]
```diff
@@ -460,10 +481,10 @@ def create_task_at_bva_dispatch_for_split_appeals(seed = Faker::Number.number(di
       appeal.tasks.where(type: "BvaDispatchTask").map(&:completed!)
     end
 
-    def create_task_at_quality_review(
+    def create_task_at_quality_review
```
Comment (seancva) [inline db/seeds/tasks.rb:484]: This method is also defined on line 549

**Review by seancva — CHANGES_REQUESTED** [review]

File: db/seeds/tasks.rb [hunk 2]
```diff
@@ -1132,6 +1154,86 @@ def create_attorney_case_review_for_legacy_appeals
         )
       end
     end
+
+    def create_vha_camo_queue_assigned
+      5.times do
+        create(:vha_document_search_task,assigned_to: VhaCamo.singleton)
```
[the remark this answers is not in the document]
Comment (pamatyatake2) [reply db/seeds/tasks.rb]: fixed

**Review by seancva — CHANGES_REQUESTED** [review]
There are no on hold appeals for the VHA Program Offices

**Review by seancva — CHANGES_REQUESTED** [review]
There is only 1 appeal in the Assigned tab for each of the VHA regional offices. There should be at least 5 in each of the tabs: Assigned, In Progress, On Hold, Completed for all of the 18 VHA Regional Offices
[Link to story](https://vajira.max.gov/browse/APPEALS-19623)

**Review by seancva — CHANGES_REQUESTED** [review]
According to this [jira ticket](https://vajira.max.gov/browse/APPEALS-19624), these users should be added to the VHA business line: VHAPOADMIN, VHAPOUSER, VISNADMIN, VISNUSER

**Review by seancva — APPROVED** [review]
LGTM

**Pull request closed without merging**